docs/backport_binary.md

```diff
@@ -0,0 +1,19 @@
+# backport binary
+Backporting binaries enables you to pin certain binaries during the
+provisioning process. This means you can setup your CoreOS cluster with
+specific versions of the `docker`, `fleet`, or `etcd` binary. This is how it
+looks like to pin fleet to the version currently provided by GiantSwarm. Simply
+set the following flags to your `yochu` executable.
+
+```
+--fleet-version=v0.11.5-gs-grpc-1 --steps=distribution,overlay,fleet
```

Comment:
Would be good to explain the `--http-endpoint` as well. Default is `https://downloads.giantswarm.io`. But this could be any other place. The expected locations for `fleet v0.11.5-gs-grpc-1` would be:

 * `https://downloads.giantswarm.io/fleet/v0.11.5-gs-grpc-1/fleetd`
 * `https://downloads.giantswarm.io/fleet/v0.11.5-gs-grpc-1/fleetctl`

Binaries will be fetched to the distribution path (defaults to: `/opt/giantswarm/bin`) and then an overlay mount is created with `/usr/bin`. So in the end your custom `fleetd` will be available through `/usr/bin/fleetd`.

See:
 * https://github.com/giantswarm/yochu/blob/master/cli/setup_cmd.go#L36
 * https://github.com/giantswarm/yochu/blob/master/steps/fleet/fleet.go#L34